oidc-consumer/src/index.ts

```diff
@@ -128,14 +128,14 @@ class OidcConsumer {
     response.redirect(authorizationURI);
   }
 
-  isOriginAllowed(url: string, allowedOrigins: any) {
-    if (allowedOrigins instanceof String || typeof allowedOrigins === "string") {
+  isRedirectUriAllowed(url: string, allowedUris: any) {
+    if (allowedUris instanceof String || typeof allowedUris === "string") {
       const { origin } = new URL(url);
```

Comment:
check against complete url, not origin

Reply:
added support for glob-urls in allowedRedirectURIs @ [1c387c5](https://github.com/getfundwave/idp-consumer/pull/2/commits/1c387c5c8bcf14b3f825a0b004017337b8b9c31d) using minimatch